skeleton/source-repo/.tekton/docker-push.yaml

```diff
@@ -43,6 +43,9 @@ spec:
   pipelineRef:
     name: docker-build-rhtap
   workspaces:
+    - name: gitops-auth
```

Comment:
I believe this is the part that was breaking the templates that we talked about in slack: https://redhat-internal.slack.com/archives/C03A062K00K/p1726005653361699

We should probably discard the commit related to it for now?

Reply:
The pipelines have been broken on main already.. The contents of [update-deployment.yaml task](https://github.com/redhat-appstudio/tssc-sample-pipelines/blob/main/pac/tasks/update-deployment.yaml) 

Even with the presence of this workspace and your temp fix branch, its ok.

Reply:
Debugged the issue:

update-deployment here needs both username and password: https://github.com/redhat-appstudio/tssc-sample-pipelines/blob/main/pac/tasks/update-deployment.yaml#L41

RHTAP 1.0.2 is broken because the secret only sets a password: https://github.com/redhat-appstudio/rhtap-installer/blob/1.0.2/chart/templates/namespace/includes/tekton/_dev_setup_task.tpl#L114

RHTAP 1.2.0 sets both username and password https://github.com/redhat-appstudio/rhtap-cli/blob/v1.2.0/charts/rhtap-dh/templates/app-namespaces/github-auth.secret.yaml#L14-L15

So their new update-deployment commit is not backwards compatible with old RHTAP.